### ISSUE TITLE
make the engine

### ISSUE THREAD

**Issue by ltmp** [Planimeter/euclid#1]
We should probably make the engine.

**Comment by wiox** [comment]
![](https://thevoiceofonedotorg.files.wordpress.com/2015/02/lck75xpca.jpeg)

**Comment by ltmp** [comment]
thanks for the help @wiox <3

**Comment by andrewmcwatters** [comment]
At this point in time, someone else besides myself would have to be the advising architect for the project.

I don't have the time to allocate to Euclid yet, so this will stay in the proj. pipeline until we can throw devs at it.

**Comment by Polkm** [comment]
I'll just point to this I guess. https://github.com/Polkm/openll bear in mind I made this project a long time ago and might not even compile anymore, but there is a shit ton of code that could be very helpful for this.

**Comment by andrewmcwatters** [comment]
Yeah, I actually had using this in mind as our reference code for Euclid.
